src/platform/user/profile/utilities/index.js

```diff
@@ -6,6 +6,8 @@ import {
 } from '../../authentication/utilities';
 import localStorage from 'platform/utilities/storage/localStorage';
 
+import { ssoKeepAliveSession } from 'platform/utilities/api/SSOhelpers';
```

Comment:
Nitpick: I think the proper camel-cased name here would be `ssoHelpers`. Alternatively, `sso-helpers` since there isn't anything exported specfically named `ssoHelpers`.